Exception/GoogleVideoTagException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * This file is part of the PrestaSitemapBundle package.
+ *
+ * (c) PrestaConcept <www.prestaconcept.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Presta\SitemapBundle\Exception;
+
+/**
+ * Exception used when limit is reached on adding tag to video
+ *
+ * @author David Epely <depely@prestaconcept.net>
+ */
+class GoogleVideoTagException extends GoogleVideoUrlTagException
```

Comment:
The new exception should not inherit from the old one, it should be the opposite, like : https://github.com/twigphp/Twig/blob/2.x/lib/Twig/Environment.php